data/os/CentOS-6.yaml

```diff
@@ -0,0 +1,2 @@
+---
+simp_snmpd::secmodel: 'usm'
```

Comment:
6 and 7 have the same data. Unless you need them split for future data, I would put this in common.